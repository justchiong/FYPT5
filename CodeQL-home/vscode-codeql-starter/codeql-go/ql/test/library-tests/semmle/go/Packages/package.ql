import go

from string path
where
  (
    path = "PackageName/v2/test" or // OK
    path = "PackageName/test" or // OK
    path = "PackageName//v//test" or // NOT OK
    path = "PackageName//v/test" or // NOT OK
    path = "PackageName/v//test" or // NOT OK
    path = "PackageName/v/asd/v2/test" or // NOT OK
    path = "PackageName/v/test" or // NOT OK
    path = "PackageName//v2//test" or // NOT OK
    path = "PackageName//v2/test" or // NOT OK
    path = "PackageName/v2//test" // NOT OK
  ) and
  path = package("PackageName", "test")
select path
