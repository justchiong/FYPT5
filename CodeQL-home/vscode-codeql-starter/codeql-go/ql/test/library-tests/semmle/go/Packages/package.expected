| PackageName/test |
| PackageName/v2/test |
